src/clj/rems/db/user_settings.clj

```diff
@@ -30,4 +51,4 @@
      {:user user
       :settings (json/generate-string
                  (merge old-settings validated))})
-    {:success (some? validated)}))
+    {:success (not (empty? validated))}))
```

Comment:
this seems to return `:success true` if e.g. language passes validation but email doesn't

Reply:
You're right. As long as at least one parameter was valid, it returns true. Currently the API is used so that only one parameter is used at a time, so the problem doesn't come up. Should the function instead give an error and do nothing if even one parameter is invalid?

Reply:
yeah